common/lib/client/auth.js

```diff
@@ -260,7 +259,13 @@ var Auth = (function() {
 						}
 					}
 					cb(null, body);
-				});
+				};
+				Logger.logAction(Logger.LOG_MICRO, 'Auth.requestToken().tokenRequestCallback', 'Sending; ' + authOptions.authUrl + '; Params: ' + JSON.stringify(authParams));
+				if(authOptions.authMethod && authOptions.authMethod.toLowerCase() === 'post') {
+					Http.postUri(rest, authOptions.authUrl, authHeaders || {}, authParams, {}, authUrlRequestCallback);
```

Comment:
I am not sure I agree.  If you wanted to add params into the body with your approach it is not possible.  However, if you wanted to add query params to the URL you could simply change the URL.  So I think what we have spec'd out provides more flexibility.